java/client/src/main/java/glide/api/commands/PubSubBaseCommands.java

```diff
@@ -40,4 +41,124 @@ public interface PubSubBaseCommands {
      * }</pre>
      */
     CompletableFuture<String> publish(GlideString message, GlideString channel);
+
+    /**
+     * Lists the currently active channels.
+     *
+     * @apiNote When in cluster mode, the command is routed to all nodes, and aggregates the response
+     *     into a single array.
+     * @see <a href="https://valkey.io/commands/pubsub-channels/">valkey.io</a> for details.
+     * @return An <code>Array</code> of all active channels.
+     * @example
+     *     <pre>{@code
+     * String[] response = client.pubsubChannels().get();
+     * assert Arrays.equals(new String[] { "channel1", "channel2" });
+     * }</pre>
+     */
+    CompletableFuture<String[]> pubsubChannels();
+
+    /**
+     * Lists the currently active channels.<br>
+     * Dislike of {@link #pubsubChannels()}, returns channel names as {@link GlideString}s.
```

Comment:
Dislike?  
```suggestion
     * Unlike of {@link #pubsubChannels()}, returns channel names as {@link GlideString}s.
```
or use "Similar to {@link }, except returns channel names as {@link }s."

Reply:
or remove